tests/Integration/Database/SchemaBuilderTest.php

```diff
@@ -26,6 +26,8 @@ public function testDropAllTables()
 
         Schema::dropAllTables();
 
+        DB::reconnect();
```

Comment:
This is only required to fixed `sqlite` and not using in-memory database. It seem using `Schema::dropAllTables()` require a new connection on GA, unable to reproduce this locally